inbox/quick-response.xml

```diff
@@ -0,0 +1,212 @@
+      </di>
+    </dl>
+  </section2>
+
+  <section2 anchor="available-action-element" topic="Available Action">
+    <p>Each available action is represented by an <tt>&lt;available-action&gt;</tt> element in the <tt>&ns;</tt> namespace.</p>
+
+<example><![CDATA[<available-action xmlns="]]>&ns;<![CDATA[" id="merge">
+  <label xml:lang="en">Merge Now!</label>
+</available-action>]]></example>
+
+    <dl>
+      <di><dt>@id</dt><dd>A string identifying the action. When selected, this id is sent in an <tt>&lt;action-selected&gt;</tt> element as part of a message stanza without any <tt>&lt;body&gt;</tt> elements.</dd></di>
+      <di><dt>&lt;label&gt;</dt><dd>Internationalized textual label for this action. One with the same <tt>@xml:lang</tt> of each <tt>&lt;body&gt;</tt> included in this <tt>&lt;message&gt;</tt>.</dd></di>
+    </dl>
+  </section2>
+
+  <section2 anchor="action-selected-element" topic="Action Selection">
+    <p>A selected action is represented by an <tt>&lt;action-selected&gt;</tt> element in the <tt>&ns;</tt> namespace.</p>
+
+<example><![CDATA[<action-selected xmlns="]]>&ns;<![CDATA[" id="merge" />]]></example>
+
+    <dl>
+      <di><dt>@id</dt><dd>The id of the selected action, as defined in the selected <tt>&lt;available-action&gt;</tt>.</dd></di>
+    </dl>
+  </section2>
+</section1>
+
+<section1 anchor="protocol" topic="Protocol">
+  <section2 anchor="sending-allowed-responses" topic="Sending a Set of Allowed Responses">
+    <p>A message with allowed responses is sent by including one or more <tt>&lt;allowed-response&gt;</tt> elements with distinct values.</p>
+
+<example><![CDATA[<message from="rootbot@example.com">
+  <body xml:lang="en">Execute `rm -rf /`? (yes/no)</body>
+  <allowed-response xmlns="]]>&ns;<![CDATA[">
+    <translation xml:lang="en" value="yes" label="Sure!" />
+  </allowed-response>
+  <allowed-response xmlns="]]>&ns;<![CDATA[">
+    <translation xml:lang="en" value="no" label="Uuuuuuh..." />
+  </allowed-response>
+</message>]]></example>
+
+    <p>A single message MUST NOT contain multiple <tt>&lt;translation&gt;</tt> elements with the same values for both <tt>@xml:lang</tt> and <tt>@value</tt>. The same applies for the combination of <tt>@xml:lang</tt> and <tt>@label</tt>.</p>
+    <p>Clients that receive a message containing allowed responses can offer UI to quickly and conveniently select one of the allowed responses.</p>
+  </section2>
+
+  <section2 anchor="selecting-a-response" topic="Selecting a Response">
+    <p>When the user selects a response, their client sends a plaintext message body containing the <tt>@value</tt> as <tt>&lt;body&gt;</tt> text. The client uses the <tt>@value</tt> that corresponds to the <tt>&lt;translation&gt;</tt> of the same language as the <tt>&lt;body&gt;</tt>.</p>
```

Comment:
I'm not actually sure what this means. It sounds like it should look at the body to pick the language to me, but the body doesn't exist yet. I assume it means the body should match the button pressed?

Reply:
Precise English phrasing might not be my strength :D I'll try to rephrase it.